source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -103,7 +103,7 @@ def get(self, request, *args, **kwargs):
             return self.list(request, *args, **kwargs)
 
         content = {
-            'detail': _("You are not authorized to list users."),
+            'non_field_errors': _("You are not authorized to list users."),
```

Comment:
Same thing here, this looks more like a permission error than a validation error.